packages/components/src/theme-type-badge/bundled-badge/style.scss

```diff
@@ -21,55 +21,54 @@
 		}
 	}
 
-	svg {
+	&__icon {
```

Comment:
Nit: can you avoid this shorthand? While it's less text, it will make finding the relevant CSS much harder using normal find/search operations. This applies here and below.

Reply:
Yep! Updated in https://github.com/Automattic/wp-calypso/pull/84483/commits/b492dc61666185a3e4e8fcd666f7745deb20f48f. Could you confirm if it matches the standards or if I should change anything else?

Basically, the commit removes the cases where it was concating class names. It keeps some `&` for multiple selectors in the same element and the nested structure though.